### ISSUE TITLE
LogHandler 

### ISSUE THREAD

**Issue by xcel3011** [zeromicro/go-zero#3529]
![image](https://github.com/zeromicro/go-zero/assets/42922046/82779721-c51b-4232-8ae6-c06496da811d)
![image](https://github.com/zeromicro/go-zero/assets/42922046/ade7852f-56db-466e-8e0e-7c4125e9d28f)
When uploading large files and using too much memory for bytes.Buffer.
Conflicting with the maxMemory param of the r.ParseMultipartForm(maxMemory).

**Comment by kevwan** [comment]
What's the size of your file? And what's your concrete request url?

**Comment by xcel3011** [comment]
I used http post methed, upload a 500Mb~1Gb form-data file, MiddlewaresConf.Log default is true, and LogHandler read all request body to a bytes.Buffer just for print log. 
I have uploaded a 1G file and used 3G of memory because slice grow.
Should LogHandler judge the Content-Type, and not read all of the request.body when multipart/form-data, or there is a document stating that Loghandler will use a lot of memory when the request.Body is too large.
